WordPress/Classes/ViewRelated/NUX/SiteCreationCategoryTableViewCells.swift

```diff
@@ -64,7 +64,7 @@ struct InstructionRow: ImmuTableRow {
     }
 }
 
-enum SiteType: String {
+enum TypeOfSite: String {
```

Comment:
This comment is associated with the class name comment in `SiteType.swift`. I'm typically reluctant to rename the in-place code unless I'm using it. What do you think?